src/main/resources/liquibase/harness/snapshot/changelogs/h2/createSequence.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+
+    <!-- Liquibase doesn't support 'dataType' attribute for mariadb, so it is removed comparing to basic changeLog-->
```

Comment:
same here need to change `mariadb` to `h2` if `datatype` isn't supported for `h2` as well